.github/pull_request_templates/feature-branch.md

```diff
@@ -0,0 +1,23 @@
+## Feature Branch: Title
+
+* Appendix doc:
+    - 
+* Link(s) to demo pages_ where this element can be viewed:
+    - 
+
+### Ready-for-merge Checklist
+
+- [ ] Expected files: all files in the merge request related to one feature request?
+- [ ] Is your branch up-to-date with the master branch?
+- [ ] Did automated tests pass?
+- [ ] Was it [browser-tested](https://docs.google.com/spreadsheets/d/1luriPYswlGcKdcbKhQfW6kxIgj7tjNjnBVP7yZ4onwA/edit#gid=1660273076)?
+- [ ] Was this feature demo'd and the design review task marked as complete?
+- [ ] Did it get a LGTM *after the last commit*? (changelog updates are exempt from this rule)
```

Comment:
We can make this automated in the repo's merging settings, sounds like a really good idea to me.

Reply:
sweet, should i leave this checkbox in place until we automate it?

Reply:
@mwcz ^

Reply:
@starryeyez024 Sorry for the delay!  I already turned it on, so you can remove the checkbox. :)